arch/x86/guest/vlapic.c

```diff
@@ -2366,7 +2366,7 @@ int apic_access_exit_handler(struct vcpu *vcpu)
 	return 0;
 }
 
-int apicv_virtualized_eoi_exit_handler(struct vcpu *vcpu)
+int virtualized_eoi_vmexit_handler(struct vcpu *vcpu)
 {
```

Comment:
Should this be eoi_vmexit_handler? or apicv_eoi_vmexit_handler?

Reply:
then veoi_vmexit_handler or eoi_vmexit_handler is fine.  We didn't use "virtualized" in other APIs.

Reply:
Got it, will update.